lib/parsec/jinja2support.py

```diff
@@ -24,12 +24,54 @@
 import os
 import sys
 from jinja2 import (
+    BaseLoader,
+    ChoiceLoader,
     Environment,
     FileSystemLoader,
-    StrictUndefined)
+    StrictUndefined,
+    TemplateNotFound)
 import cylc.flags
 
 
+class PyModuleLoader(BaseLoader):
+    """Load python module as Jinja2 template.
+
+    This loader piggybacks on the jinja import mechanism and
+    returns an empty template that exports module's namespace."""
+
+    # no source access for this loader
+    has_source_access = False
+
+    def __init__(self, prefix='__python__'):
+        self._templates = {}
+        # prefix that can be used to avoid name collisions with template files
+        self._python_namespace_prefix = prefix + '.'
+
+    def load(self, environment, name, glbs=None):
```

Comment:
Please add:
* A doc string.
* A comment on why we are good to ignore Codacy here.